views/profile.ejs

```diff
@@ -49,7 +49,7 @@
                 <% for (var i = 0; i < user.interests.length; i++) { %>
 
                     <% if (i % 4 == 0 && user.interests[i] !== '') { %>
-                        <div class="tag red-icon"><%=  %></div>
+                        <div class="tag red-icon"><%= user.interests[i] %></div>
                     <% } else if (i % 3 == 1 && user.interests[i] !== '') { %>
```

Comment:
I thought there was some error issue regarding trimming the input.. Went totally unnoticed. 
Thanks!